Templates/project_collection/README.md

```diff
@@ -0,0 +1,212 @@
+# Instructions for filling out Project Collection Template
+The project and data collection template contains all the information necessary
+to add a new project and/or data collection to our inventory. There are three
+sheets in this Excel document: `Data collection project`, `Data collection`,
+and `values`. The first two sheets are to be filled out with the appropriate
+information about your project and data collection. The third sheet contains
+controlled vocabulary and should not be modified.
+
+If you have any questions, please feel free to email
+[data.curation@alleninstitute.org](mailto:data.curation@alleninstitute.org)
+so we can assist you with adding your metadata to the project inventory.
+
+
+Please refer to the following descriptions of values when filling `project_collection_template.xlsx`:
+
+## Data collection project
+This sheet contains information about your data collection project
+
+#### **Project** - *project name* 
+  * This is a unique human-readable identifier (i.e. handle) used to keep track 
+    of your project
+  * By convention, we prefer to use names in `snake_case_with_proj` as a suffix
+  * If you are not sure what project name to use, please email 
+    [data.curation@alleninstitute.org](mailto:data.curation@alleninstitute.org), 
+    and we will help you choose one
+  
+#### **is part of** - *subprogram*
+  * This is the subprogram that your project is part of (often a grant name)
+  * This field is controlled by the vocabulary in the `subprogram` column on `values` sheet
+  * If you don't see your subprogram on that list, please email 
+    [data.curation@alleninstitute.org](mailto:data.curation@alleninstitute.org), 
+    and we will add it
+    
+#### **is highlighted by** - *URL*
+  * This is a hyperlink to something related to your project
+  * A landing page with a general overview of your project is best
+  * If you don't have a landing page, you could include a DOI for a flagship paper,
+    or a link to the repository to where the data for this project are stored
+  
+#### **is funded by** - *grant*
+  * This is the grant that is funds your project
+  * This field is controlled by the vocabulary in the `grant` column on `values` sheet
+  * If you don't see your grant on that list, please email 
+    [data.curation@alleninstitute.org](mailto:data.curation@alleninstitute.org), 
+    and we will add it
```

Comment:
No action needed, just a comment:
Should we create a separate grant ingest template? How are you ingesting all the grant details?

Reply:
I feel like all of this should be handled by the website long-term. There is no process for adding new grants yet. Any tabular format (csv is nice) with similar fields to this template could be easily adapted to ingest any type of `Entity` that we want, including grants.